streaming/src/main/java/org/apache/spark/streaming/status/api/v1/BatchStatus.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.streaming.status.api.v1;
```

Comment:
allright,
I understand the problem now.

in order to merge with another plan(SPARK-18085) in the furture
streaming api may need to support history in the furture,
and thus need to use /api/v1/applications/:id/:attempt/streaming as endpoint

in order to use /api/v1/applications/:id/:attempt/streaming as end point
someone will need to implement a hooking mechanism to "mount" the streaming api to the applications resource

am I correct?